GoogleMapsApi.Test/UnixTimeConverterTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Reflection;
+using NUnit.Framework;
+using GoogleMapsApi.Engine;
+
+namespace GoogleMapsApi.Test
+{
+    [TestFixture]
+    public class UnixTimeConverterTest
+    {
+        /// <summary>
+        /// This test verifies that the DateTimeToUnixTimestamp prdouces the expected Unix timestamps
+        /// </summary>
+        [Test]
+        public void DateTimeToUnixTimestamp_Test()
+        {
+            Assert.AreEqual(UnixTimeConverter.DateTimeToUnixTimestamp(new DateTime(2016, 4, 4, 12, 0, 0, DateTimeKind.Local)), 1459764000);
```

Comment:
Please test one thing in each test so when it fails you have a better understanding of what failed.